platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+   proposal.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections. Diagrams and other files
+   may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+9. Mark the as ready for review
+10. Update the status of the PR in
+    the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for inmaterial changes to the proposal can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of inmaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of content, etc. If any material changes are needed, the PR must be closed, and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
+
+The magnitude of the numeric value of the vote indicates the strength of the sentiment behind the vote.
+
++1 votes can also be expressed by approving the PR. -1 votes can be expressed by "Requesting Changes" on the PR and
+must be accompanied by a comment explaining the reason for the veto.
+
+---
+
+## Acceptance of A Proposal
+
+A proposal becomes `Accepted` when the following criteria have been fulfilled:
+
+1. The proposal has been in the Voting state for at least 3 business days. Business days are defined as SwirldsLabs
+   working days and excludes weekends and company holidays.
+2. The proposal must have at least three +1 votes from platform code owners.
+3. The proposal must not have any -1 votes from platform code owners or architects.
+
+Once accepted, the proposal must be moved to the Accepted status in
+the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board and implementation of the
+proposal may begin.
```

Comment:
Is the proposal not merged? I dislike the idea that proposals should exist only in PRs and not in the `main` branch of the repo.

Reply:
I assume proposals are merged and we have a history of them over time. Each proposal should represent the delta from design state S to S', and the documentation should represent the current design at any given time.

Reply:
forgot to include this, will add